addons/refuel/script_component.hpp

```diff
@@ -29,3 +30,9 @@
     _weaponSelect = _unit getVariable QGVAR(selectedWeaponOnRefuel); \
     _unit selectWeapon _weaponSelect; \
     _unit setVariable [QGVAR(selectedWeaponOnRefuel), nil];
+
+#ifdef FAST_PROGRESSBARS
+    #define TIME_PROGRESSBAR(X) ((X) * 0.075)
+#else
+    #define TIME_PROGRESSBAR(X) (X)
+#endif
```

Comment:
`TIME_PROGRESSBAR` is already in rearm
Rearm, Refuel are the main ones with some long progress bars that make testing tedious.

Might actually cause problems on some things like repacking and medical surgery, not sure.